q2_assembly/megahit/megahit.py

```diff
@@ -189,15 +198,17 @@ def _assemble_megahit(
             "then all must be explicitly set."
         )
 
-    kwargs = {k: v for k, v in locals().items() if k not in ["seqs"]}
+    kwargs = {k: v for k, v in locals().items() if k not in ["seqs", "co_assemble"]}
     common_args = _process_common_input_params(
         processing_func=_process_megahit_arg, params=kwargs
     )
 
-    return assemble_megahit_helper(seqs=seqs, common_args=common_args)
+    return assemble_megahit_helper(
+        seqs=seqs, common_args=common_args, co_assemble=co_assemble
+    )
 
 
-def assemble_megahit_helper(seqs, common_args) -> ContigSequencesDirFmt:
+def assemble_megahit_helper(seqs, common_args, co_assemble) -> ContigSequencesDirFmt:
```

Comment:
It would be nice if the order of args here could be: "seqs, coassemble, common_args". `common_args` is just a list of all the other arguments that should be passed to megahit so it would be nice if this can always be the last argument.